.travis.yml

```diff
@@ -8,16 +8,25 @@ matrix:
     - rust: nightly
 
 install:
-  - |
-      pip install 'travis-cargo<0.2' --user &&
-      export PATH=$HOME/.local/bin:$PATH
+  - rustup toolchain install nightly-2019-10-13 || exit 1;
```

Comment:
How about using `minimal` profile?
https://blog.rust-lang.org/2019/10/15/Rustup-1.20.0.html

Reply:
Looks nice. I updated the scripts.